meta-mender-core/recipes-mender/mender-client/mender-client-cpp.inc

```diff
@@ -0,0 +1,17 @@
+require mender-client.inc
+
+RPROVIDES:${PN} = "mender-client"
+
+DESCRIPTION = "Mender tool for doing OTA software updates."
+HOMEPAGE = "https://mender.io"
+
+SYSTEMD_SERVICE:${PN} = "mender-updated.service mender-authd.service"
+
+MENDER_CLIENT ?= "mender-updated"
+
+FILES:${PN} += "\
```

Comment:
Same, `${PN}` needs split.

Reply:
While we're on the topic, should we split these further, like have sub-overrides like `mender-dbus` etc?

Reply:
`mender-dbus` is not necessary I believe, since dbus is now required to run mender (you can't communicate with mender-auth otherwise), but in general we should consider splitting where appropriate.